src/hooks/useAllServices.ts

```diff
@@ -95,103 +100,114 @@ const filterAllServicesSections = (allServicesLinks: AllServicesSection[], filte
     return acc;
   }, []);
 };
+
+const findNavItems = (
+  items: (string | AllServicesLink | AllServicesGroup)[],
+  availableLinks: { id?: string; title?: string; items: AvailableLinks }[]
+): (AllServicesLink | AllServicesGroup)[] =>
+  items
+    .map((item) => {
+      if (isAllServicesGroup(item as AllServicesGroup)) {
```

Comment:
Same type guard casting. You won't need the additional typecast below if the type guard does not use it.